Sources/MapboxDirections/TollPrice.swift

```diff
@@ -0,0 +1,136 @@
+import Foundation
+import Turf
+
+/**
+ :nodoc:
+ Information about toll payment method.
+ */
+public struct PaymentMethod: Hashable, Equatable {
+    /**
+     Method identifier.
+     */
+    public let identifier: String
```

Comment:
Interesting choice to make the identifier a member of this struct. If we were following the usual extensible enumeration pattern, this struct would only contain `static let`s of type `String`. The more structured approach you’ve taken is more flexible, but JSON doesn’t allow an object’s keys to be objects, so this flexibility would be unused.